### ISSUE TITLE
Add learing rate finder

### ISSUE THREAD

**Issue by sotte** [GRAAL-Research/poutyne#16]
Are you interested in a learning rate finder as described in 
https://arxiv.org/abs/1506.01186 section 3.3?
I ported my LR finder to PyToune. It still needs some work but it's quite handy!

**Comment by freud14** [comment]
It seems nice. However, isn't it more suitable to integrate that directly into PyTorch with their LR schedulers? I mean, if the PyTorch team is not interested, I would happily be interested for PyToune.

**Comment by freud14** [comment]
Or maybe I misunderstood the method?

**Comment by freud14** [comment]
Do we still need this issue?

**Comment by sotte** [comment]
I would implement this (or change the implementation that I currently use) after #17 is merged.

**Comment by freud14** [comment]
@sotte Are you still working on that?

**Comment by sotte** [comment]
I kept using my initial implementation. I'm not planning on rewriting it. So I would close this issue.

Thanks for this great tool!
